test/src/api-action-creator-tests.js

```diff
@@ -236,6 +236,27 @@ describe('APIActionCreators', function () {
         r.abort();
     });
 
+    it('should call handleFailure on a network error', function (done) {
```

Comment:
How is this test testing a network error? It is testing an HTTP 404 error right? I think we already have sufficient tests around abort and normal HTTP failures.

Reply:
@jhorwit2 You're completely right yea.
I'm not actually sure if we can fake a network error in a test now that I think about it.
Would you like the test changes removed for merging just?

Reply:
@rjmcneill yeah, lets remove the test. The only thing I could think of to test this is set the base url to a random IP that'll timeout but that's super hacky.

Reply:
@jhorwit2 removed the test and updated PR 👍